internal/domain/auth.go

```diff
@@ -12,3 +12,7 @@ type AuthUseCase interface {
 	Logout(ctx context.Context, input *input.RefreshInput) error
 	RefreshToken(ctx context.Context, input *input.RefreshInput) (*output.TokenOutput, error)
 }
+
+type AuthRepository interface {
+	ExistByRefreshToken(ctx context.Context, refreshToken string) (bool, error)
```

Comment:
만약 저희가 이 BlackListRepository가 refreshToken을 관리한다는 것을 알면 굳이 한번 더 명시할 필요는 없을 것 같아요!
```suggestion
	Exists(ctx context.Context, refreshToken string) (bool, error)
```